src/entity/Fruits.java

```diff
@@ -1,19 +1,24 @@
 package entity;
 
+import java.util.Objects;
+
 public class Fruits {
     private int id;
     private String name;
     private double price;
     private int quantity;
     private String countryOfOrigin;
+    private String color;
```

Comment:
Literówka, Color -> colour 
```suggestion
    private String colour;
```